run_aprime.bash

```diff
@@ -0,0 +1,325 @@
+export obs_iceareaSH=$obs_seaicedir/IceArea_timeseries/iceAreaSH_climo.nc
+export obs_icevolNH=$obs_seaicedir/PIOMAS/PIOMASvolume_monthly_climo.nc
+export obs_icevolSH=none
+
+##############################################################################
+
+######### USER SHOULD NOT NEED TO CHANGE ANYTHING HERE ONWARDS ###############
+
+export coupled_diags_home=$PWD
+
+# LOAD THE MACHINE-SPECIFIC ANACONDA-2.7 ENVIRONMENT
+if [ $machname6 == "edison" ]; then # NERSC
+  source /opt/modules/default/init/bash  
+  module unload python
+  module unload python_base
+  module use $projdir/software/modulefiles/all
+  module load python/anaconda-2.7-acme
+fi
+if [ $machname4 == "rhea" ] || [ $machname5 == "titan" ]; then # OLCF
+  module unload python
+  module use $projdir/pwolfram/modulefiles/all
+  module load python/anaconda-2.7-climate
+fi
+if [ $machname5 == "aims4" ] || [ $machname5 == "acme1" ]; then # LLNL (aims4/acme1)
+  echo
+  echo "Need to set/run the following:"
+  echo "  export PATH=/usr/local/anaconda2/bin:$PATH"
+  echo "  source activate ACME-UNIFIED   # (on aims4)"
+  echo "  source activate ACME_UNIFIED   # (on acme1)"
+  echo
+fi
+if [ $machname4 == "wolf" ]; then # LANL
+  module unload python
+  module use $projdir/modulefiles/all
+  module load python/anaconda-2.7-climate
+fi
+
+# PUT THE PROVIDED CASE INFORMATION IN CSH ARRAYS TO FACILITATE READING BY OTHER SCRIPTS
+./bash_scripts/setup.bash
+
+# RUN DIAGNOSTICS
+if [ $generate_atm_diags -eq 1 ]; then
+  # Check whether requested files for computing climatologies are available
+  rpointer_file=${test_archive_dir}/${test_casename}/run/rpointer.atm
+  year_max=`grep -m 1 -Eo '\<[0-9]{4}\>' ${rpointer_file} | awk '{print $1-1}'`
+  if [ $test_end_yr_climo -le $year_max ]; then
+    #
+    if ! $run_batch_script; then
+      ./bash_scripts/aprime_atm_diags.bash
+    else
+      if [ $machname6 == "edison" ]; then # NERSC
+        sbatch ./bash_scripts/batch_atm.NERSC.bash
+      else
+        echo
+        echo "Batch jobs not supported on current machine"
+        echo "Please set $run_batch_script to false"
+        echo
+      fi
+    fi
+    #
+    atm_status=$? 
+  else
+    echo
+    echo "Requested test_end_yr_climo is larger than the maximum simulation year. Exiting atm diagnostics..."
+    echo "Please set test_end_yr_climo <= ${year_max}"
+    echo
+    atm_status=0
+  fi
+else
+  atm_status=0
+fi
+
+if [ $generate_ocnice_diags -eq 1 ]; then
+  # Check whether requested files for computing climatologies are available
+  rpointer_file=${test_archive_dir}/${test_casename}/run/rpointer.ocn
+  year_max=`grep -m 1 -Eo '\<[0-9]{4}\>' ${rpointer_file} | awk '{print $1-1}'`
+  if [ ${test_end_yr_climo} -le ${year_max} ]; then
+    #
+    if ! $run_batch_script; then
+      ./bash_scripts/aprime_ocnice_diags.bash
+    else
+      if [ $machname6 == "edison" ]; then # NERSC
+        sed -i "s/nodes=.*/nodes=$mpas_analysis_tasks/" ./bash_scripts/batch_ocnice.NERSC.bash
```

Comment:
I don't think modifying `bash_scripts/batch_ocnice.NERSC.bash` in place is a good idea, since this is a file in the repository that might get committed with the changes.  Instead, `sed` should be creating a temporary copy of the bash script with the modification which is then run.  Probably the temporary script will need to be in the output directory (just like the config file) to prevent conflicts if analysis is run simultaneously on multiple simulations.